src/js/classes.js

```diff
@@ -151,7 +151,7 @@ Classes.CLASSES = {
           },
           ultimate: {
             name: 'Hellhound',
-            effect: 'Spawn a Hellhound Token with (Courage Lvl) DMG and (Courage Lvl) Max HP. Hellhound Token is treated as an ally User (cannot acquire or hold Gear). There can only be 1 Hellhound Token at a time',
+            effect: 'Spawn a Hellhound Token with (Courage Lvl) DMG and (Courage Lvl) Max HP. Hellhound Token is treated as an ally User (cannot acquire or hold Gear). If Grimwulf Kaiser targeted for an attack, Hellhound Token is attacked instead. There can only be 1 Hellhound Token at a time',
```

Comment:
If Grimwulf Kaiser targeted for an attack
　↓
If Grimwulf Kaiser is targeted for an attack